modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminNavigationDisplayContext.java

```diff
@@ -72,10 +76,19 @@ public KBAdminNavigationDisplayContext(
 
 		_kbArticleURLHelper = new KBArticleURLHelper(
 			renderRequest, renderResponse);
+
+		_liferayPortletRequest = PortalUtil.getLiferayPortletRequest(
+			(PortletRequest)_httpServletRequest.getAttribute(
+				JavaConstants.JAVAX_PORTLET_REQUEST));
+
 		_liferayPortletResponse = LiferayPortletUtil.getLiferayPortletResponse(
 			renderResponse);
+
 		_themeDisplay = (ThemeDisplay)_httpServletRequest.getAttribute(
 			WebKeys.THEME_DISPLAY);
+
+		_kbDropdownItemsProvider = new KBDropdownItemsProvider(
+			_liferayPortletRequest, _liferayPortletResponse);
```

Comment:
This should be moved just before the initialization of `_themeDisplay` so that they're sorted alphabetically/by dependency.

Reply:
done!